### PR TITLE
add clusterID label for managedCluster

### PR DESCRIPTION
**Pull request by zhiweiyin318**
task: https://github.com/open-cluster-management/backlog/issues/5021

### CODE REVIEW

File: pkg/klusterlet/clusterinfo/clusterinfo_controller.go [hunk 1]
```diff
@@ -531,24 +534,53 @@ func (r *ClusterInfoReconciler) getDistributionInfo() (clusterv1beta1.Distributi
...
-		// status.desired.version
-		DesiredVersion: desiredVersion,
-		// true when the first status.conditions element with type === Failing has status === True
-		UpgradeFailed: upgradeFailed,
-		// the value of the version property from each element in status.availableUpdates
-		AvailableUpdates: availableVersions,
+	return ocpDistributionInfo, nil
+}
+
+func (r *ClusterInfoReconciler) getOCPClusterID() (string, error) {
```
Comment (qiujian16) [inline pkg/klusterlet/clusterinfo/clusterinfo_controller.go:546]: seems to me that getOCPDistributionInfo already gives us the clusterID. How about we also return cluster ID from getOCPDistributionInfo?

**Review by qiujian16 — COMMENTED** [review]
I think  we need an e2e test for this

File: pkg/klusterlet/clusterinfo/clusterinfo_controller.go (hunk 1 above)
Comment (zhiweiyin318) [reply pkg/klusterlet/clusterinfo/clusterinfo_controller.go]: done. using  `getOCPDistributionInfo` can reduce one API call.

**Comment by zhiweiyin318** [comment]
> I think we need an e2e test for this

done

**Pull request merged**